packages/components/src/buttons/src/Button.module.css

```diff
@@ -0,0 +1,277 @@
+    --hop-button-component-upsell-color-hover: var(--hop-upsell-text-hover);
+    --hop-button-component-upsell-color-pressed: var(--hop-upsell-text-active);
+    --hop-button-component-upsell-color-disabled: var(--hop-neutral-text-disabled);
+    --hop-button-component-upsell-background-color: var(--hop-upsell-surface);
+    --hop-button-component-upsell-background-color-hover: var(--hop-upsell-surface-hover);
+    --hop-button-component-upsell-background-color-pressed: var(--hop-upsell-surface-active);
+    --hop-button-component-upsell-background-color-disabled: var(--hop-neutral-surface-disabled);
+    --hop-button-component-upsell-border: 0.0625rem solid var(--hop-upsell-surface); /* same color as the background */
+    --hop-button-component-upsell-border-hover: 0.0625rem solid var(--hop-upsell-surface-hover); /* same color as the background */
+    --hop-button-component-upsell-border-pressed: 0.0625rem solid var(--hop-upsell-border-active);
+    --hop-button-component-upsell-border-disabled: 0;
+
+    /* Negative */
+    --hop-button-component-negative-color: var(--hop-danger-text-strong);
+    --hop-button-component-negative-color-hover: var(--hop-danger-text-strong-hover);
+    --hop-button-component-negative-color-pressed: var(--hop-danger-text-active);
+    --hop-button-component-negative-color-disabled: var(--hop-neutral-text-disabled);
+    --hop-button-component-negative-background-color: var(--hop-danger-surface-strong);
+    --hop-button-component-negative-background-color-hover: var(--hop-danger-surface-hover);
+    --hop-button-component-negative-background-color-pressed: var(--hop-danger-surface-active);
+    --hop-button-component-negative-background-color-disabled: var(--hop-neutral-surface-disabled);
+    --hop-button-component-negative-border: 0.0625rem solid var(--hop-danger-surface-strong); /* same color as the background */
+    --hop-button-component-negative-border-hover: 0.0625rem solid var(--hop-danger-surface-hover); /* same color as the background */
+    --hop-button-component-negative-border-pressed: 0.0625rem solid var(--hop-danger-border-active);
+    --hop-button-component-negative-border-disabled: 0;
+
+    /* Internal variable */
+    --inline-size: max-content;
+    --block-size: var(--hop-button-component-md-height);
+    --background-color: var(--hop-primary-surface-strong);
+    --color: var(--hop-primary-text-strong);
+    --border: 0;
+
+    /** styles */
+    cursor: pointer;
+    display: grid;
+    grid-template-areas:"start-icon content end-icon";
+    grid-template-columns: auto min-content auto;
+    column-gap: var(--hop-space-inline-xs);
+    box-sizing: border-box;
+    position: relative;
+    padding: var(--hop-button-component-padding);
+    border-radius: var(--hop-button-component-border-radius);
+    outline: none;
+    outline-offset: 0.125rem;
+    transition:/* TODO: create transition for components */
+        color 0.2s cubic-bezier(0.22, 0.61, 0.36, 1),
```

Comment:
where does these comes from? (timing and easings)

Reply:
Copied from an example of buttons to test. 
I created an issue for this: https://github.com/gsoft-inc/wl-hopper/issues/181